test/sql/function/string/hex.test

```diff
@@ -0,0 +1,95 @@
+# name: test/sql/function/string/hex.test
+# description: Test the to_hex/from_hex function
+# group: [string]
+
+statement ok
+PRAGMA enable_verification
+
+query I
+SELECT to_hex('duckdb');
+----
+6475636B6462
+
+query I
+SELECT from_hex('6475636B6462');
+----
+duckdb
+
+# Test at least one byte
+query I
+SELECT to_hex(0::TINYINT);
+----
+00
+
+# Test skip leading zero
+query I
+SELECT to_hex(8388609::INT);
+----
+800001
+
+query I
+SELECT to_hex(32767::SMALLINT);
+----
+7FFF
+
+query I
+SELECT to_hex(-32767::SMALLINT);
+----
+8001
+
+# Test hugeint
+query I
+SELECT to_hex(18446744073709551616::HUGEINT);
+----
+010000000000000000
+
+query I
+SELECT to_hex(170141183460469231731687303715884105727);
+----
+7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+
+query I
+SELECT from_hex('0110');
+----
+\x01\x10
+
+
+# Test '110' == '0110'
+query I
+SELECT from_hex('110');
+----
+\x01\x10
+
+# Test NULL
+statement ok
+CREATE TABLE test (x INT);
+
+statement ok
+INSERT INTO test VALUES (1), (NULL), (2);
+
+query I
+SELECT to_hex(x) FROM test
+----
+01
+NULL
+02
+
+statement ok
+DROP TABLE test
+
+statement ok
+CREATE TABLE test (x VARCHAR);
+
+statement ok
+INSERT INTO test VALUES ('01'), (NULL), ('02');
+
+query I
+SELECT from_hex(x) FROM test
```

Comment:
Could we add some tests with unexpected values in the input, and very long input values?